cc-pk-service/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# This is where you define the image for
+# Materiaalitori service setup.
+
+# Start by copying the server stub to this directory.
+
+
+# First line is provided for your convenience.
+# This line fetches the Node Docker image with tag "16"
+# from Docker Hub.
+# We use version 16 as the mongodb dependency has requirements
+FROM node:latest
+
+
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN npm install
+
+RUN npm install -g typescript
+
+RUN npm install express
```

Comment:
Is this necessary? Isn't it already installed with npm install?

Reply:
updated. express installation

Reply:
i am not sure about typescript part because i faced issue and install manually typescript.